modules/core/src/main/scala/org/tessellation/modules/Services.scala

```diff
@@ -21,11 +20,10 @@ object Services {
   ): F[Services[F]] =
     for {
       metrics <- Metrics.make[F]
-      healthcheck = HealthCheck.make[F]
       stateChannelRunner <- StateChannelRunner.make[F](queues.stateChannelOutput)
     } yield
       new Services[F](
-        healthcheck = healthcheck,
+        healthcheck = sdkServices.healthCheck,
```

Comment:
I didn't move it intentionally, because healthcheck will be rewritten. but that's fine either ;)